azure/common/deploy-stages.yml

```diff
@@ -1,12 +1,6 @@
 parameters:
   - name: apigee_deployments
     type: object
-  - name: secret_file_ids
-    type: object
-  - name: secret_ids
```

Comment:
Won't this cause errors as the pr and release template expect those parameters?

Reply:
Its still passed through. I just tried it outside the environment params. Having them here was causing it to error when passing in as a param on the environment